lib/msal-common/src/authority/Authority.ts

```diff
@@ -152,6 +153,7 @@ export abstract class Authority {
      */
     public async resolveEndpointsAsync(): Promise<void> {
         const openIdConfigEndpoint = await this.getOpenIdConfigurationEndpointAsync();
-        this.tenantDiscoveryResponse = await this.discoverEndpoints(openIdConfigEndpoint);
+        const response = await this.discoverEndpoints(openIdConfigEndpoint);
+        this.tenantDiscoveryResponse = response.body;
```

Comment:
Same comment about fetch in msal browser

Reply:
The response from fetch is wrapped in a NetworkResponse object, defined in msal-common. This is part of the throttling PR